src/components/Clock.tsx

```diff
@@ -0,0 +1,58 @@
+import { Component } from 'react';
+
+type State = {
+  today: Date;
+};
+
+type Props = {
+  clockName: string;
+};
+
+export class Clock extends Component<Props, State> {
+  state = {
+    today: new Date(),
+  };
+
+  timerClockTimeId = 0;
+
+  componentDidMount(): void {
+    this.timerClockTimeId = window.setInterval(() => {
+      this.setState({ today: new Date() });
+
+      window.console.info(
+        this.state.today.toUTCString().slice(-12, -4),
```

Comment:
It can be a helper function you can reuse.

Reply:
same here, create helper func